lib/usecases/base/base_use_case.dart

```diff
@@ -1,3 +1,5 @@
+import '../../api/exception/network_exceptions.dart';
```

Comment:
Why do we need this import? 🤔

Reply:
@ryan-conway It's used by `use_case_result.dart`. I know it's weird, but when importing it by the IDE, it was added here. I'm not sure why. I also tried to move it to `use_case_result` too. But it doesn't work.

Reply:
@ryan-conway You can try to fix it on your end. 😄